tools/local-test/scripts/test-by-ibcmd.sh

```
@@ -1,45 +1,105 @@
+
+if [[ ! -d "${EDT_PATH}" ]]; then
+    >&2 echo "Не найдена инсталяция ${EDT_PATH}"
+    exit 1
+fi
+
+if [[ ! -d "$ONEC_PATH" ]]; then
+    >&2 echo "Не найдена инсталяция ${ONEC_PATH}"
+    exit 1
+fi
+
+export WORK_PATH=$(realpath $SCRIPTPATH/../workpath)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)